robotoff/cli/batch.py

```diff
@@ -47,8 +48,10 @@ def batch_annotate(
             i += 1
             print("Insight %d" % i)
             print(
-                "Add label {} to https://fr.openfoodfacts.org/produit/{}"
-                "".format(insight.data, insight.barcode)
+                "Add label {} to {}/produit/{}"
```

Comment:
actually, we need to swap produit as well by product
or do something like fr-en.openfoodfacts.org/product/ ( or even better world.openfoodfacts.org/product/)